app/views/customers/registrations/new.html.erb

```diff
@@ -1,29 +1,31 @@
-<h2>Sign up</h2>
-
 <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
   <%= devise_error_messages! %>
 
-  <div class="field">
-    <%= f.label :email %><br />
-    <%= f.email_field :email, autofocus: true %>
-  </div>
-
-  <div class="field">
-    <%= f.label :password %>
-    <% if @minimum_password_length %>
-    <em>(<%= @minimum_password_length %> characters minimum)</em>
-    <% end %><br />
-    <%= f.password_field :password, autocomplete: "off" %>
-  </div>
-
-  <div class="field">
-    <%= f.label :password_confirmation %><br />
-    <%= f.password_field :password_confirmation, autocomplete: "off" %>
-  </div>
-
-  <div class="actions">
-    <%= f.submit "Sign up" %>
-  </div>
+<div class="container">
+  <div class="row">  
+    <div class="col-md-4 col-sm-4 col-md-offset-4 col-sm-offset-4">
+      <div class="login-form">
+        <h3>Sign Up</h3>
+        <form role="form">
+        <div class="form-group">
+         <label>Email address</label>
```

Comment:
All these labels need for attributes for screen readers and stuff. We need to follow as much accessibility concerns as we can with this app